server/risk_scores/risk_assessment.py

```diff
@@ -126,23 +131,38 @@ def get_current_risk_score(form: submit_schema.Form):
     return percent_of_max
 
 
+required_email_form_fields = {'staff_name', 'client_primary'}
+
+
 def get_risk_assessment(form: submit_schema.Form, timeframe) -> RiskAssessment:
     total_risk_score = get_current_risk_score(form)
-    + get_previous_risk_score(form, timeframe)
+    +get_previous_risk_score(form, timeframe)
 
-    for i, (max_percent, assessment) in enumerate(assessment_ranges):
+    for (max_percent, assessment) in assessment_ranges:
         if total_risk_score <= max_percent:
-            # check if risk score is above the required minimum to send an email 
-            if i >= minimum_email_score_index and credentials.PYTHON_ENV != "development":
-                email_high_risk_alert(form.dict()) # TODO: make async
+            email_dict = {
+                k: v
+                for k, v in form.dict().items()
+                if k in required_email_form_fields
+            }
+            email_dict.update({
+                'risk_score': assessment.value,
+                'risk_score_raw': total_risk_score
+            })
+            email_high_risk_alert(email_dict)
             return assessment
     else:
         return RiskAssessment.UNDEFINED
 
 
-def email_high_risk_alert(form_values: dict):
-    form_values = (
-        f"<b>{field}</b>: {value}" for field, value in form_values.items())
+html_template_filename = "/.template.html"
```

Comment:
Is there a reason why we start the filename with the `.`?

Reply:
just thought it looks cool ngl, lemme fix